<launch>

	<!-- This launche requires to launch the RoboCup world included in
		CSUROS (https://github.com/IntelligentRoboticsLabs/CSUROS)
		using: roslaunch csuro_robots sim_robocup.launch
	-->

	<node pkg="bica_examples" type="yellow_net_detector_node" name="yellow_net_detector" output="screen"/>
	<node pkg="bica_examples" type="ball_detector_node" name="ball_detector"  output="screen"/>
	<node pkg="bica_examples" type="blue_net_detector_node" name="blue_net_detector"  output="screen"/>
	<node pkg="bica_examples" type="ballnet_follower_node" name="ballnet_follower"  output="screen"/>
	<node pkg="bica" type="launcher" name="launch_ball_follower" args="ballnet_follower"  output="screen"/>
	<node pkg="rqt_gui" type="rqt_gui" name="rqt_gui" args="-s bica_rqt_graph"/>

</launch>
